src/lib/evil/unposix/unistd.h

```diff
@@ -1,10 +1,18 @@
-#ifndef UNISTD_H
-#define UNISTD_H
+#ifndef UNPOSIX_UNISTD_H
+#define UNPOSIX_UNISTD_H
 
 #include "unimplemented.h"
 
+#include <corecrt.h>
+#include <BaseTsd.h>
+#include <sys/types.h>
+#include <../ucrt/sys/types.h>
```

Comment:
What is the difference between those two? Isn't the sys/types.h ao unposix "implementation"?

Reply:
Yes, it is and this include is indeed duplicated (since sys/types should call /ucrt/sys/types.h)

Reply:
Should be fixed now.